add_me_to_your_PATH/julia_pod

```diff
@@ -96,6 +96,7 @@ fi
 # sanitize to be valid dns entry
 RUNID="$(echo $RUNID | sed -e 's/[^a-zA-Z0-9]\+/-/g')-$(date +%a%Hh%Mm%S)"
 RUNID="$(echo ${RUNID##*(-)} | tr '[:upper:]' '[:lower:]')"
+RUNID=${RUNID}-$RANDOM
```

Comment:
Using `generateName` in the k8s manifest would be best. Using `openssl rand -hex 3` is slightly better than using `$RANDOM`